src/target/linux.rs

```diff
@@ -84,13 +89,35 @@ impl UnixTarget for LinuxTarget {
     fn pid(&self) -> Pid {
         self.pid
     }
+
+    fn unpause(&self) -> Result<nix::sys::wait::WaitStatus, Box<dyn std::error::Error>> {
+        ptrace::cont(self.pid(), None)?;
```

Comment:
If there was a call to `restore_breakpoint` previously you need to step one instruction and set the breakpoint instruction again before continuing. Otherwise breakpoints can only be hit a single time.